### PR TITLE
[TRN-156] Sort out docusaurus configs and logos

### PR DESCRIPTION
**Pull request by vldolot**
Summary:
 - changed `trn-docs` logo
 - changed favicon
 - default theme set to dark mode
 - some docusaurus config changes

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -43,9 +48,9 @@ module.exports = {
         showReadingTime: true,
         editUrl: 'https://github.com/polkadot-js/docs/edit/master/',
       },
-      theme: {
-        customCss: require.resolve('./src/css/custom.css'),
-      }
+      // theme: {
+      //   customCss: require.resolve('./src/css/custom.css'),
+      // }
```
Comment (vldolot) [inline docusaurus.config.js:53]: Might still need these commented config for the branding task/s

File: docusaurus.config.js [hunk 2]
```diff
@@ -1,21 +1,26 @@
-const BASE_URL = process.env.NODE_ENV === 'development' ? '/' : '/docs/';
+const BASE_URL = '/';
 
 module.exports = {
-  title: 'polkadot{.js}',
-  tagline: 'Polkadot does JavaScript',
-  url: 'https://polkadot.js.org',
+  title: 'The Root Network',
+  tagline: 'The Root Network docs',
```
Comment (ken-centrality) [inline docusaurus.config.js:5]:
```suggestion
  tagline: 'The Root Network documentation',
```

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "polkadot-js",
-  "version": "0.0.270",
+  "name": "trn-docs",
```
Comment (ken-centrality) [inline package.json:2]:
```suggestion
  "name": "@therootnetwork/docs",
```

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
...
+  "version": "0.0.1",
```
Comment (ken-centrality) [inline package.json:3]:
```suggestion
  "version": "1.0.0",
```

**Comment by ken-centrality** [comment]
- Remove App Wallet link
- Update the Search box color to be blue

**Review by ken-centrality — APPROVED** [review]

**Pull request merged**